### PR TITLE
Send userId and dirty bucket name

### PR DESCRIPTION
**Pull request by MancunianSam**
This is to allow the antivirus checks to work with both the existing
dirty bucket and the new cloudfront dirty bucket so we pass the bucket
that triggered the notification to the antivirus lambda.

I've also removed cognitoId from the messages sent to the checksum
lambda and file format lambda as neither lambda uses them.

I've added userId alongside cognitoId so the antivirus doesn't break
while deploying this change.

This PR needs to deploy first, then
https://github.com/nationalarchives/tdr-antivirus/pull/37 and when
that's merged, cognitoId can be removed from the antivirus output in
this lambda.

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala [hunk 1]
```diff
@@ -77,10 +76,12 @@ class Lambda {
...
             new File(s"$prefix/$writeDirectory").mkdirs()
             val writePath = s"$efsRootLocation/$consignmentId/$originalPath"
+            val dirtyBucketName = record.getS3.getBucket.getName
             val s3Response = fileUtils.writeFileFromS3(writePath, fileId, record, s3).map(_ => {
-              fileFormatSendMessage(DownloadOutput(cognitoId, consignmentId, fileId, originalPath).asJson.noSpaces)
-              antivirusSendMessage(DownloadOutput(cognitoId, consignmentId, fileId, originalPath).asJson.noSpaces)
-              checksumSendMessage(DownloadOutput(cognitoId, consignmentId, fileId, originalPath).asJson.noSpaces)
+              val output = DownloadOutput(consignmentId, fileId, originalPath).asJson.noSpaces
+              fileFormatSendMessage(output)
+              antivirusSendMessage(AntivirusDownloadOutput(consignmentId, fileId, originalPath, userId, userId, dirtyBucketName).asJson.noSpaces)
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala:83]: Have got the param `userId` twice, is that correct? Should one of them be `cognitoId`

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment.

File: src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala (hunk 1 above)
Comment (MancunianSam) [reply src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala]: The cognitoId and userId are the same value in the old and new buckets, i.e. the first part of the s3 path so technically this is right but it's a bit weird so I've added in another variable for cognitoId with the same value and a comment to remind me to remove it later.

**Review by TomJKing — APPROVED** [review]

File: src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala (hunk 1 above)
Comment (TomJKing) [reply src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala]: Makes sense.

**Pull request merged**